app/controllers/conversions/voluntary/projects_controller.rb

```diff
@@ -50,7 +50,8 @@ def create
       :note_body,
       :assigned_to_regional_caseworker_team,
       :directive_academy_order,
-      :sponsor_trust_required
+      :sponsor_trust_required,
+      :region
```

Comment:
Do we need to permit this if it never comes in via the form or is not massed assigned?

Reply:
Good point, don't think we need to